File: 백준/Bronze/28224. Final Price/Final Price.cs
using System;

class Program {
    static void Main() {
        int n = int.Parse(Console.ReadLine());
        int total = 0;

        for (int i = 0; i < n; i++) {
            int a = int.Parse(Console.ReadLine());
            total += a;
        }

        Console.WriteLine(total);
    }
}

File: 백준/Bronze/28224. Final Price/Final Price.vb
Module Program
    Sub Main()
        Dim n As Integer = Integer.Parse(Console.ReadLine())
        Dim total As Integer = 0
        
        For i As Integer = 0 To n - 1
            Dim a As Integer = Integer.Parse(Console.ReadLine())
            total += a
        Next
        
        Console.WriteLine(total)
    End Sub
End Module